%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ELIFE ARTICLE TEMPLATE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% PREAMBLE 
\documentclass[9pt,lineno]{elife}
% Use the onehalfspacing option for 1.5 line spacing
% Use the doublespacing option for 2.0 line spacing
% Please note that these options may affect formatting.
% Additionally, the use of the \newcommand function should be limited.

\usepackage{lipsum} % Required to insert dummy text
\usepackage[version=4]{mhchem}
\usepackage{siunitx}
\usepackage{mathtools}
\DeclareSIUnit\Molar{M}


%% Include all macros below
\newcommand{\sticky}{proglobular~}
\newcommand{\dSNPs}{dSNPs~}
\newcommand{\nSNPs}{nSNPs~}
\newcommand{\dSNP}{dSNP~}
\newcommand{\nSNP}{nSNP
\newcommand{\refa}{{\rm R}}
\newcommand{\alta}{{\rm A}}
\newcommand{\nSNPset}{\mathcal{N}}
\newcommand{\dSNPset}{\mathcal{D}}
\newcommand{\hydrochar}{hydrophobicity class}
\newcommand{\chargechar}{charge class}
\newcommand{\grace}[1]{\textcolor{red}{#1}}
\newcommand{\ruchi}[1]{\textcolor{blue}{#1}}
\newcommand{\matt}[1]{\textcolor{purple}{Matt says: #1}}
\newcommand{\cmax}{H_{\rm max}}
\newcommand{\hydrothresh}{H^{\star}}
\newcommand{\cmin}{\hydrothresh}
\newcommand{\Lminp}{L_{\rm min}^p}
\newcommand{\Lminh}{L_{\rm min}^h}
\newcommand{\Lmax}{L^{\star}}
\newcommand{\Lmin}{L_{\rm min}}
\newcommand{\het}{\Theta}
\newcounter{includefigs}
\setcounter{includefigs}{1}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ARTICLE SETUP
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title{Supporting information for Contiguously-hydrophobic sequences are functionally significant throughout the human exome}
%\title{Contiguous hydrophobic residues confer disease risk throughout the human exome}

%\title{Hydrophobicity based sequence blobulation approach captures functional modularity: disease associated mutations are enriched in hydrophobic blobs}

\author[1\authfn{3}]{Ruchi Lohia}
\author[2\authfn{1}]{Matthew E.B. Hansen}
\author[1,3\authfn{1}*]{Grace Brannigan}
\affil[1]{Center for Computational and Integrative Biology, Rutgers University, Camden, NJ, USA}
\affil[2]{Department of Genetics, University of Pennsylvania, Philadelphia, PA, USA}
\affil[3]{Department of Physics, Rutgers University, Camden, NJ, USA}

\corr{* grace.brannigan@rutgers.edu}{GB}

\contrib[\authfn{1}]{These authors contributed equally to this work}

\presentadd[\authfn{3}]{Stanley Institute for Cognitive Genomics, Cold Spring Harbor Laboratory, USA}
%\presentadd[\authfn{4}]{Department, Institute, Country}
% \presentadd[\authfn{5}]{eLife Sciences editorial Office, eLife Sciences, Cambridge, United Kingdom}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ARTICLE START
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\renewcommand{\thepage}{S\arabic{page}}  
\renewcommand{\thesection}{S\arabic{section}}   
\renewcommand{\thetable}{S\arabic{table}}   
\renewcommand{\figurename}{}
\renewcommand{\thefigure}{Fig S\arabic{figure}}

\maketitle
   
\subsection*{SI Tables}

\textbf{SI_Dataset_1.txt:} This is the base blobulation data per SNP, for all nSNPs and dSNPs, for 21 different hydrophobicity thresholds $H^*=0,0.05,0.1,\dots,1$. This is a tab-delimited text file (gzipped). Rows correspond to each unique pairing of SNP and hydrophobicity threshold $H^*$. The columns are: a unique variant ID in the format <UniProt_protein_ID>:<reference amino acid><residue number><alternative amino acid> (``ID''); the UniProt ID of the protein containing the variant (``UniProt_Protein_ID''); the amino acid polymorphism (``AA_Polymorphism''); the disease-association label, with ``N'' for nSNP and ``D'' for dSNP (``Assoc''); the hydrophobicity threshold $H^*$ used (``Hydrophobicity_Min''); the blob type for the sequence with the reference amino acid, with values ``h'', ``p'', and ``s'' (``Blob_Type_REF''); the blob type for the sequence with the alternative amino acid (``Blob_Type_ALT''); the length of the blob surrounding the variant for teh sequence with the reference amino acid (``L_REF''); the length of the blob surrounding the variant for the sequence with the alternative amino acid (``L_ALT''); 
\hfill

\textbf{SI_Dataset_2.xslx:} Enrichment of dSNPs relative to nSNPs in h blobs for 2D binned hydrophobicity cutoffs and blob lengths. This is the blobulation data used for Figure 2 of the Main Text. Contains three sheets. Each sheet has the same format. Rows correspond to SNP counts each each unique 2D bin of blob parameters $\{H^*,L\}$. The columns are: the blob length $L$ (``Blob length''); the blob hydrophobicity threshold $H^*$ (``Hydrophobicity threshold $H^*$''); the number of dSNPs in blobs with these blob parameters (``Number of dSNPs''); the number of nSNPs in blobs with these blob parameters (``Number of nSNPs''); the total number of SNPs in blobs with these blob parameters (``Total number of SNPs''); the enrichment of dSNPs in blobs with these parameters (``Enrichment of dSNPs''); and the Binomial test p-value for the enrichment (``p value''). Sheet1: all regions, corresponding to Figure 2d,h. Sheet2: solvated regions, corresponding to Figure 2e,i. Sheet3: transmembrane regions, corresponding to Figure 2f,j. 
\hfill

\textbf{SI_Dataset_3.txt:} Population frequency data per SNP. This is the data used for Figure 3 and SI Figure 1, and is a merge of SI_Dataset_1.txt and gnomAD SNP frequency data based on variant ID listed in the ``ID'' column. It is a tab-delimited text file (gzipped). Each row is the data corresponding to a UniProt variant with gnomAD frequency data blobulated by the listed hydrophobicity threshold. The columns are: a unique variant ID in the format <UniProt_protein_ID>:<reference amino acid><residue number><alternative amino acid> (``ID''); the UniProt ID of the protein containing the variant (``UniProt_Protein_ID''); the amino acid polymorphism (``AA_Polymorphism''); the disease-association label, with ``N'' for nSNP and ``D'' for dSNP (``Assoc''); the hydrophobicity threshold $H^*$ used (``Hydrophobicity_Min''); the blob type for the sequence with the reference amino acid, with values ``h'', ``p'', and ``s'' (``Blob_Type_REF''); the blob type for the sequence with the alternative amino acid (``Blob_Type_ALT''); the length of the blob surrounding the variant for teh sequence with the reference amino acid (``L_REF''); the length of the blob surrounding the variant for the sequence with the alternative amino acid (``L_ALT''); the variant rsid (``dbSNP''); the protein's gene name, in HUGO nomenclature (``Gene_Name''); the UniProt ID of the variant (``Uniprot_Var_ID''); the chromosome the variant is in (``Chrom''); the base pair position of the variant on the chromosome, in GRCh37 coordinates (``Pos''); the reference allele nucleotide (``Ref''); the alternative allele nucleotide (``Alt''); the number of alternative alleles in the gnomAD non-Finnish European frequency data (``N_Alt_gnomad:nfe''); the total number of alleles in the gnomAD non-Finnish European frequency data (``N_Tot_gnomad:nfe''); the frequency of the alternative allele in the gnomAD non-Finnish European frequency data (``Freq_Alt_gnomad:nfe''); the minor allele frequency in the gnomAD non-Finnish European frequency data (``MAF_gnomad:nfe''); the number of alternative alleles in the gnomAD East Asian frequency data (``N_Alt_gnomad:eas''); the total number of alleles in the gnomAD East Asian frequency data (``N_Tot_gnomad:eas''); the frequency of the alternative allele in the gnomAD East Asian frequency data (``Freq_Alt_gnomad:eas''); the minor allele frequency in the gnomAD East Asian frequency data (``MAF_gnomad:eas''); the expected population heterozygosity in the non-Finnish European cohort (``Het:nfe''); and the expected population heterozygosity in the East Asian cohort (``Het:eas'').
\hfill

\textbf{SI_Dataset_4.xlsx: GO enrichment for nSNPs and dSNPs in highly hydrophobic h-blobs} 
Gene set pathway and ontology enrichment test results using g:Profiler (Raudvere {\it et al}, 2019). Contains two sheets. Each sheet has the same format. Rows correspond to significantly enriched GO terms, where significance is based on the g:SCSS multiple-testing corrected p-value $\leq 0.05$. The columns are: the gene ontology database source (``source''); the GO term (``term_name''); GO term ID (``term_id''); the g:SCS multiple-testing corrected p-value (``adjusted_p_value''); the negative log10 of the p-value (``negative_log10_of_adjusted_p_value''); the number of proteins labeled with the GO term (``term_size''); the number of proteins in the query set (``query_size''); the number of query proteins labeled with the GO term (``intersection_size''); the fraction of the query set in the intersection (``fquery''); the fraction of the proteins labeled with the GO term in the intersection (``fterm''); the geometric mean of the two fterm and fquery (``fgeom''); the total number of background variants used (``effective_domain_size''); and the UniProt IDs for the proteins in the intersection (``intersections''). Sheet1: GO results for target proteins containing nSNPs with maximum hydrophobicity $\cmax\geq 0.75$ ($n=635$), compared to a background of all proteins containing an nSNP ($n=10,406$). Sheet2: GO results for target proteins containing dSNPs with maximum hydrophobicity $\cmax\geq 0.75$ ($n=179$), compared to a background of all proteins containing an nSNP ($n=1,803$).  
\hfill

\subsection*{SI Figures}

\textbf{SI Figure 1: Heterozygosity of SNPs in East Asians as a function of the SNP maximum blob hydrophobicity} 
Similar to Figure 3 but for the gnomAD East Asian population cohort.
\hfill

\begin{figure}[!ht]
\includegraphics[scale=0.5,width=\textwidth,trim={0 0cm 0 0cm},clip]{./figures/SI_Figure_1.pdf}
\caption{{\bf Heterozygosity of SNPs in East Asians as a function of blob hydrophobicity}. Similar to Figure 2 but for the gnomAD East Asian population cohort.}
\label{S1} 
\end{figure}
\clearpage


%\nocitep{*} % This command displays all refs in the bib file. PLEASE DELETE IT BEFORE YOU SUBMIT YOUR MANUSCRIPT!


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% APPENDICES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\end{document}









